test/unittests/parser_test.cpp

```diff
@@ -9,6 +9,14 @@ static const auto functype_void_to_void = from_hex("600000");
 static const auto functype_i32i64_to_i32 = from_hex("60027f7e017f");
 static const auto functype_i32_to_void = from_hex("60017f00");
 
+namespace
+{
+bytes push_section(uint8_t id, const bytes& section)
```

Comment:
`bytes_view section` wasn't working (needed to be converted), so this seemed simpler.